docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Quickstart Walkthrough
+id: quickstart-walkthrough
+---
+The following will walk you through the manual process of creating an overlay network with your own PKI. These steps are the same steps that take place behind the scenes when running the OpenZiti Quickstart’s `expressInstall` function.
+
+## General Environment Setup
+
+### Declare Variables
+
+Some global variables used throughout the overlay setup.
+
+- `ZITI_HOME` is the directory where your network files will be stored, the directory will be created if it does not exist already.
+- `ZITI_BIN_DIR` is the directory where the ziti binary will be downloaded and extracted
+- `ZITI_PWD` is the password that will be used when logging into the controller as the default user (admin).
```

Comment:
I think you should add `ZITI_USER` too. also is `ZITI_HOSTNAME` actually ___used___ anywhere? it doesn't show up in constants.go and I couldn't find any references to it???

Reply:
also maybe remove the 'your' and replace it with "the directory `expressInstall` will use to create the necessary files"

Reply:
ZITI_HOSTNAME was used in this doc because I used your video as a reference, I thought you wanted me to essentially write a doc that followed the same process as you did in the video (which is also the quickstart process).

I'll update this to ZITI_NETWORK